docs/source/references/config_files/project/base_yml.rst

```diff
@@ -161,8 +161,32 @@ these settings have to be their default values as shown in the
 
       :port:
         | The port on which the service can be reached
+        | ``integer``
+
+  :communicationStore:
+    | Attributes pertaining to Redis container storing communications
+    | ``{object}`` which has the following subkeys
+
+      :host:
+        | The IP/name of the host machine or container running this service
         | ``string``
 
+      :port:
+        | The port on which the service can be reached
+        | ``integer``
+
+  :verificationStore:
+    | Attributes pertaining to Redis container storing verifications
```

Comment:
I don't know what this is for but I feel it can be explained more clearly. @pradumangoyal can explain.

Reply:
They are used to store and verify the verification tokens.